src/main/java/com/lorepo/icplayer/client/module/text/GapWidget.java

```diff
@@ -105,6 +110,43 @@ public void onKeyUp(KeyUpEvent event) {
 						listener.onValueEdited(gapInfo.getId(), getText());
 						listener.onUserAction(gapInfo.getId(), getText());
 					}
+					if (gapInfo.isNumericOnly()) {
```

Comment:
Standardowo, fajnie byłoby mieć test pod to, żeby lepiej wiedzieć co się wpasowuje lub nie.